lib/absinthe/type/built_ins/introspection.ex

```diff
@@ -279,6 +279,8 @@ defmodule Absinthe.Type.BuiltIns.Introspection do
 
         _, %{schema: schema, source: %{default_value: value, type: type}} ->
           case Absinthe.Schema.lookup_type(schema, type, unwrap: true) do
+            %Absinthe.Type.Enum{values_by_internal_value: values} ->
+              {:ok, values[value].name}
```

Comment:
I'm not sure how to handle when the default value does not map to any enum value.

Reply:
@fsabe we should probably try to add a schema validation rule that ensures all default values are valid. Let's handle that separately.